README.md

```diff
@@ -156,34 +160,40 @@ DDL_STATEMENTS=$(
   | jq -R -s -c 'split(";")'
 )
 ```
+
 Finally, to create the database:
+
 ```sh
 curl -sS --request POST \
   "localhost:9020/v1/projects/$PROJECT_ID/instances/$INSTANCE_ID/databases" \
   --header 'Accept: application/json' \
   --header 'Content-Type: application/json' \
   --data "{\"createStatement\":\"CREATE DATABASE \`$DATABASE_ID\`\",\"extraStatements\":$DDL_STATEMENTS}"
 ```
+
 Note that, again, you may set `DATABASE_ID` to your liking. Make sure that the `database_url` config variable reflects your choice of project name, instance name, and database name (i.e. it should be of the format `spanner://projects/<your project ID here>/instances/<your instance ID here>/databases/<your database ID here>`).
 
 To run an application server that points to the local Spanner emulator:
+
 ```sh
 SYNC_SYNCSTORAGE__SPANNER_EMULATOR_HOST=localhost:9010 make run_spanner
 ```
 
 ### Running via Docker
-This requires access to the mozilla-rust-sdk which is now available at `/vendor/mozilla-rust-sdk`.
+
+This requires access to [Google Cloud Rust (raw)](https://github.com/mozilla-services/google-cloud-rust-raw/) crate. Please note that due to interdependencies, you will need to ensure that `grpcio` and `protobuf` match the version used by `google-cloud-rust-raw`.
```

Comment:
Hrm, I get a 404 not found when following this link. Is the `raw` crate private? I see the path to `google-cloud-rust` works fine here: https://github.com/mozilla-services/google-cloud-rust. However the listing for `google-cloud-rust-raw` is active in `lib.rs`: https://lib.rs/crates/google-cloud-rust-raw

Reply:
No, I pasted in the wrong link. It should be https://crates.io/crates/google-cloud-rust-raw/ (sigh)